src/js/components/CosmosErrorMessage.js

```diff
@@ -41,25 +46,54 @@ class CosmosErrorMessage extends React.Component {
   }
 
   getDetails() {
-    let {data} = this.props.error;
-    if (!data || !data.errors) {
-      // No errors
+    let {error} = this.props;
+    if (!error) {
+      return null;
+    }
+
+    // Return early if we have some well-known or an unknown type
+    if (typeof error === 'string') {
+      return [error];
+    } else if (typeof error !== 'object') {
```

Comment:
Not sure what you are referring to, but this:
```js
if (typeof error === 'string') {
  return [error];
}
if (typeof error !== 'object') {
  return null;
}
```
is the same as this:
```js
if (typeof error === 'string') {
  return [error];
} else if (typeof error !== 'object') {
  return null;
}
```

If it is a string it will return in the first block, if it is not a string and not an object it will return in the second block. If it is none of these it will continue in both cases